src/char/int_party.c

```diff
@@ -317,7 +317,7 @@ struct party_data* inter_party_search_partyname(const char *const str)
 int inter_party_check_exp_share(struct party_data *const p)
 {
 	nullpo_ret(p);
-	return (p->party.count < 2 || p->max_lv - p->min_lv <= party_share_level);
+	return (p->party.count < 2 || (int)(p->max_lv - p->min_lv) <= party_share_level);
```

Comment:
I believe the fix to this is not this simple. I see in various places that `p->max_lv` is set to 0, and `p->min_lv` to `UINT_MAX`, and then someone had the bright idea to perform this scary unsigned difference in this function (this might date back to the Japanese Athena).

I'd prefer to change the type of `p->max_lv` and `p->min_lv` to a safer `int`, to get rid of this and many other possible underflows.